src/workerd/api/streams/queue.c++

```diff
@@ -249,23 +248,20 @@ void ValueQueue::visitForGc(jsg::GcVisitor& visitor) {}
 
 #pragma region ByteQueue::ReadRequest
 
-namespace {
-void maybeInvalidateByobRequest(kj::Maybe<ByteQueue::ByobRequest&>& req) {
-  KJ_IF_SOME(byobRequest, req) {
+void ByteQueue::ReadRequest::maybeInvalidateByobRequest() {
+  KJ_IF_SOME(byobRequest, byobReadRequest) {
     byobRequest.invalidate();
     // The call to byobRequest->invalidate() should have cleared the reference.
```

Comment:
Nit: comment seems ambiguous now that the `KJ_ASSERT` it was documenting is gone. Maybe s/cleared the reference/cleared byobReadRequest/?

Edit: I see you delete this function later, though.